app/connectors/PertaxConnector.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import cats.data.EitherT
+import config.ApplicationConfig
+import models.PertaxApiResponse
+import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse, UpstreamErrorResponse}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class PertaxConnector @Inject() (httpClient: HttpClient, applicationConfig: ApplicationConfig)(implicit
+  ec: ExecutionContext
+) {
+
+  private val baseUrl = applicationConfig.pertaxHost
+
+  def pertaxAuth(nino: String)(implicit hc: HeaderCarrier): EitherT[Future, UpstreamErrorResponse, PertaxApiResponse] =
+    EitherT(
+      httpClient.GET[Either[UpstreamErrorResponse, PertaxApiResponse]](s"$baseUrl/pertax/$nino/authorise")
```

Comment:
The Accept header is missing. See:https://github.com/hmrc/benefits-frontend/blob/1dfe9d49d968bc042bb561fc362cfa97d4f132dc/app/connectors/paye/PertaxConnector.scala#L25

Reply:
Done